module/stats.py

```diff
@@ -0,0 +1,22 @@
+"""/stats command"""
+from telegram import Update
+from telegram.ext import CallbackContext
+from module.shared import SELECT, STATS_MESSAGE
+from module.create_connection import connect_and_execute
+
+def stats(update: Update, context: CallbackContext) -> None:
+    """called by /stats
+        It returns the number of books that are now on sale
+
+        Args:
+            update: update event
+            context: context passed by the handler"""
+
+    query: str = "SELECT COUNT(*) FROM Market"
+    chat_id: int = update.effective_chat.id
+    params = ()
```

Comment:
why are you giving an empty tuple here?
if it is really necessary you could write params=() directly in the connect_and_execute function, but I hope that it is an optional parameter and you could omit it

Reply:
Yes, effectively I could pass the empty tuple directly in the function